procedures/unit-testing-tracing.ipf

```diff
@@ -172,9 +172,22 @@ static Function/S PreCheckProcedures(string procWinList)
 	return outList
 End
 
+/// @brief Returns the traced procedures. Do not modify these!
+threadsafe static Function/WAVE GetTracedProcedures()
+	TUFXOP_GetStorage/N="IUTF_Traced_Procedures" wvStorage
+	if(V_flag)
+		UTF_Reporting#UTF_PrintStatusMessage("Error: Cannot get IUTF_Traced_ProcedureNames storage for traced procedure names")
```

Comment:
The error message here is a copy of the one from GetTracedProcedureNames and should probably be different.
How about naming this GetTracedProcedurePaths to be more distinct from the other global.

While the procedure names must be available for all threads the paths don't have to be. It is easier to debug if stored in the IUTF Package folder.

Reply:
Fixed it.